extensions/wikia/ArticleVideo/scripts/article-video.js

```diff
@@ -98,36 +83,25 @@ require(['wikia.window', 'wikia.onScroll', 'wikia.tracker', 'ooyalaVideo'], func
 			});
 		}
 
-		function updatePlayerControls(waitForTransition) {
-			ooyalaVideoController.hideControls();
-			if (waitForTransition && transitionEndEventName) {
-				$videoContainer.on(transitionEndEventName, function () {
-					ooyalaVideoController.sizeChanged();
-					ooyalaVideoController.showControls();
-				});
-			} else {
-				ooyalaVideoController.sizeChanged();
+		function updateOoyalaSize() {
+			window.dispatchEvent(new Event('resize'));
```

Comment:
This is to artificially trigger some Ooyala event, right? Please explain it in a comment.